### PR TITLE
Resolving merge conflicts on USDOT ODE PR

### PR DESCRIPTION
**Pull request by dmccoystephenson**
## Problem
There are merge conflicts on https://github.com/usdot-jpo-ode/jpo-ode/pull/547.

## Solution
The merge conflicts have been resolved by merging USDOT's develop branch into CDOT dev.

## Testing
Unit tests have been verified to pass with these changes.

### CODE REVIEW

File: jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/OdeProperties.java [hunk 1]
```diff
@@ -119,6 +119,7 @@ public class OdeProperties implements EnvironmentAware {
    private Set<String> kafkaTopicsDisabledSet = new HashSet<>();
 
    // BSM
+   private final String BSM_START_FLAG = "0014";
```
Comment (mwodahl) [inline jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/OdeProperties.java:122]: These strings were moved into the UperUtil class in #78 . Are they currently used anywhere from the OdeProperties class? If not I don't believe we should add these back in.

Comment (dmccoystephenson) [reply jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/OdeProperties.java]: Thanks for this! These have been removed & the unit tests have been verified to be passing still.

**Review by mwodahl — APPROVED** [review]
Looks good!

**Review by payneBrandon — APPROVED** [review]
lgtm!

**Pull request merged**